### PR TITLE
Create governance.md

### PR DESCRIPTION
**Pull request by caitlinthefirst**
New Governance doc for Gatsby website. Related to issue [#11805](https://github.com/gatsbyjs/gatsby/issues/11805).

### CODE REVIEW

File: docs/contributing/governance.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Governance
```
Comment (LekoArts) [inline docs/contributing/governance.md:1]:
```suggestion
```

You'll need to remove this and instead add the doc to the sidebar links list, then the title of that will be "Governance"

File: docs/contributing/governance.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+It’s important to note that Gatsby’s governance model is a work in progress that will continue to grow and evolve over time. When new governance-related issues arise, Gatsby’s leadership and Core Team will do their best to include the community and anyone directly involved in the issue in the decision-making process.
+
+## How are decisions made about Gatsby OSS?
+
+### Contributing roles
+
+The following groups are the primary categories of Gatsby OSS contributors.
+
+- [Maintainers](https://github.com/orgs/gatsbyjs/teams/maintainers) - The outstanding community of contributors who help make Gatsby awesome. These are people who’ve had a Gatsby pull request merged and adhere to the [Gatsby Code of Conduct](https://www.gatsbyjs.org/contributing/code-of-conduct/).
+- [Admins](https://github.com/orgs/gatsbyjs/teams/admin/members) - The official team members of Gatsby, Inc. (sometimes referred to as the "inkteam" including the Gatsby Core Team who maintain Gatsby OSS.
```
Comment (LekoArts) [inline docs/contributing/governance.md:22]:
```suggestion
- [Admins](https://github.com/orgs/gatsbyjs/teams/admin/members) - The official team members of Gatsby, Inc. (sometimes referred to as the "inkteam") including the Gatsby Core Team who maintain Gatsby OSS.
```

File: docs/contributing/governance.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+- [Community Maintainers Meeting](https://www.gatsbyjs.org/contributing/community/#community-maintainers-meeting)
+- [Pair programming sessions](https://www.gatsbyjs.org/contributing/pair-programming/)
+
+## Engage with the Gatsby team
+
+Here are some other ways to communicate with us:
+
+- [@askgatsbyjs Twitter](https://twitter.com/AskGatsbyJS) - specifically for your questions and feedback
+- [@gatsbyjs Twitter](https://twitter.com/gatsbyjs) - for Gatsby news, updates, and community engagement
+- [Gatsby on Github](https://github.com/gatsbyjs)
```
Comment (LekoArts) [inline docs/contributing/governance.md:57]:
```suggestion
- [Gatsby on GitHub](https://github.com/gatsbyjs)
```

File: docs/contributing/governance.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Governance
+
+## Introduction
+
+Gatsby.js is an [open source](https://opensource.org/) project supported by [Gatsby, Inc.](https://www.gatsbyjs.com/about/) employees and an active community of contributors.
+
+[Gatsby’s community and corporate values](https://www.gatsbyjs.org/docs/gatsby-core-philosophy/#gatsbys-community-philosophy) include working in the open and setting and managing clear expectations. Establishing a governance model helps us live those values and will help support active and productive communication within the Gatsby community and between the community and the company.
```
Comment (Ekwuno) [inline docs/contributing/governance.md:7]:
```suggestion
[Gatsby’s community and corporate values](/docs/gatsby-core-philosophy/#gatsbys-community-philosophy) include working in the open and setting and managing clear expectations. Establishing a governance model helps us live those values and will help support active and productive communication within the Gatsby community and between the community and the company.
```

File: docs/contributing/governance.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+It’s important to note that Gatsby’s governance model is a work in progress that will continue to grow and evolve over time. When new governance-related issues arise, Gatsby’s leadership and Core Team will do their best to include the community and anyone directly involved in the issue in the decision-making process.
+
+## How are decisions made about Gatsby OSS?
+
+### Contributing roles
+
+The following groups are the primary categories of Gatsby OSS contributors.
+
+- __Outside Contributors__ - The outstanding community of contributors who help make Gatsby awesome. These are people who’ve had a Gatsby pull request merged and adhere to the [Gatsby Code of Conduct](https://www.gatsbyjs.org/contributing/code-of-conduct/).
```
Comment (Ekwuno) [inline docs/contributing/governance.md:21]: Same suggestion as above
```suggestion
- __Outside Contributors__ - The outstanding community of contributors who help make Gatsby awesome. These are people who’ve had a Gatsby pull request merged and adhere to the [Gatsby Code of Conduct](/docs/contributing/code-of-conduct/).
```

File: docs/contributing/governance.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+## Guidance and documentation for contributors
+
+Here are some relevant docs and resources for contributors:
+
+- [Contributing to Gatsby.js](https://www.gatsbyjs.org/contributing/)
+- [How to contribute](https://www.gatsbyjs.org/contributing/how-to-contribute/)
+- [How to file an issue](https://www.gatsbyjs.org/contributing/how-to-file-an-issue/)
+- [Request-for-comment (RFC) process](https://www.gatsbyjs.org/contributing/rfc-process/)
+- [Community Maintainers Meeting](https://www.gatsbyjs.org/contributing/community/#community-maintainers-meeting)
+- [Pair programming sessions](https://www.gatsbyjs.org/contributing/pair-programming/)
```
Comment (Ekwuno) [inline docs/contributing/governance.md:57]:
```suggestion
- [Contributing to Gatsby.js](/docs/contributing)
- [How to contribute](/docs/contributing/how-to-contribute/)
- [How to file an issue](/docs/contributing/how-to-file-an-issue/)
- [Request-for-comment (RFC) process](/docs/contributing/rfc-process/)
- [Community Maintainers Meeting](/docs/contributing/community/#community-maintainers-meeting)
- [Pair programming sessions](/docs/contributing/pair-programming/)
```

File: docs/contributing/governance.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+Our pull request review process is outlined in our [Managing Pull Requests doc](https://www.gatsbyjs.org/contributing/managing-pull-requests/).
+
+If you have any questions about an issue or PR, you can add a comment to the issue/PR on GitHub or reach out to us via one of the options in the "Engage with the Gatsby Team" section below.
+
+## Code of Conduct
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+[Read the full code of conduct](https://www.gatsbyjs.org/contributing/code-of-conduct/)
```
Comment (Ekwuno) [inline docs/contributing/governance.md:46]:
```suggestion
[Read the full code of conduct](/docs/contributing/code-of-conduct/)
```

File: docs/contributing/governance.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+### Feature Requests
+
+If you’d like to request or propose a Gatsby feature (or a change to an existing feature), the best way to start is to submit it as an issue via GitHub. The Core Team will then review your submission and decide whether or not to put it through our formal “Request for Comment” process.
+
+### Community PR and Issue Review Process
+
+Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](https://www.gatsbyjs.org/contributing/triaging-github-issues/).
+
+Our pull request review process is outlined in our [Managing Pull Requests doc](https://www.gatsbyjs.org/contributing/managing-pull-requests/).
```
Comment (Ekwuno) [inline docs/contributing/governance.md:38]:
```suggestion
Our pull request review process is outlined in our [Managing Pull Requests doc](/docs/contributing/managing-pull-requests/).
```

File: docs/contributing/governance.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+- __[Admins](https://github.com/orgs/gatsbyjs/teams/admin/members)__ - The official team members of Gatsby, Inc. (sometimes referred to as the "inkteam") including the Gatsby Core Team who maintain Gatsby OSS.
+
+### Decision-Making Process
+
+Decisions about Gatsby open-source software are ultimately made by Gatsby, Inc. leadership and the Core Team. We believe that founding organizations built around open source software have a responsibility to build strong businesses to sustain those open source tools and the community that depends on them, which will sometimes require us to reserve high-demand tools and features for Gatsby commercial products. That said, we take our responsibility to the community very seriously and carefully consider community needs and concerns when determining our product roadmaps.
+
+_If you’d like to learn more about how we approach the relationship between our commercial and open source work, check out this blog post by Gatsby co-founder Sam Bhagwat: [Founding Organizations: Creating a Company That Sustains Our Open-Source Community](https://www.gatsbyjs.org/blog/2020-02-11-founding-organizations/#reach-skip-nav)._
```
Comment (Ekwuno) [inline docs/contributing/governance.md:28]:
```suggestion
_If you’d like to learn more about how we approach the relationship between our commercial and open source work, check out this blog post by Gatsby co-founder Sam Bhagwat: [Founding Organizations: Creating a Company That Sustains Our Open-Source Community](/blog/2020-02-11-founding-organizations/#reach-skip-nav)._
```

File: docs/contributing/governance.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+### Feature Requests
+
+If you’d like to request or propose a Gatsby feature (or a change to an existing feature), the best way to start is to submit it as an issue via GitHub. The Core Team will then review your submission and decide whether or not to put it through our formal “Request for Comment” process.
+
+### Community PR and Issue Review Process
+
+Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](https://www.gatsbyjs.org/contributing/triaging-github-issues/).
```
Comment (Ekwuno) [inline docs/contributing/governance.md:36]:
```suggestion
Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](/docs/contributing/triaging-github-issues/).
```

File: docs/contributing/governance.md (hunk 6 above)
Comment (muescha) [reply docs/contributing/governance.md]: note: the `/contributing*` are not under the `/docs*`

Comment (muescha) [reply docs/contributing/governance.md]: > note: the `/contributing*` are not under the `/docs*`

not fixed :(

**Comment by muescha** [comment]
still 404 on `/docs/contributing`
they exists under `/contributing`

**Comment by muescha** [comment]
new PR: 

- #26370 `fix(docs): governance model -> fix 404 on internal links`

**Pull request merged**